techniques/fileDistribution/copyGitFile/2.1/copyFileFromSharedFolder.st

```diff
@@ -130,7 +130,7 @@ bundle agent download_from_shared_folder
       # Suid and sgid may be setted only if the option to set it recursively, and recursion level is 1, or we are managing a file
       # If it's a directory, without exclusion
       "${copyfile[${index}][destination]}"
-        copy_from    => rudder_copy_from("&SHARED_FILES_FOLDER&/${copyfile[${index}][name]}", "${server_info.cfserved}", "${copyfile[${index}][compare_method]}", "false", "false", "${copyfile[${index}][purge]}"),
+        copy_from    => rudder_copy_from("&SHARED_FILES_FOLDER&/${copyfile[${index}][name]}", "${server_info.policy_server}", "${copyfile[${index}][compare_method]}", "false", "false", "${copyfile[${index}][purge]}"),
```

Comment:
this would break if the server is updated and the techniques are synchronized to another environment (while system techniques are not)

Reply:
😿

Reply:
Maybe we could already define the value in 4.1 so people migrating would not have issues if they are in latest 4.1 !

Reply:
https://github.com/Normation/rudder-techniques/pull/1253 in 4.1

Reply:
Alexis' concern has been treated in #1253